nova_plugin/tests/resources/test-networks-relationships-blueprint.yaml

```diff
@@ -0,0 +1,82 @@
+
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml
+  - plugin.yaml
```

Comment:
(for future reference: turns out this is explicitly done by the `@workflow_test` decorator)